examples/basic/lines/arcs.py

```diff
@@ -1,3 +1,14 @@
+''' An arc graph using pre-defined data points. This example
+demonstrates the use of the ``arc`` method to make a graph
+by drawing three arcs of defined radius, start and
+end angles at a specified point.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.arc
+    :refs: :ref:`ug_basic_lines_arcs
```

Comment:
missing trailing backtick causing docs build to fail
```suggestion
    :refs: :ref:`ug_basic_lines_arcs`
```